processing/src/test/java/org/apache/druid/query/aggregation/firstlast/last/StringLastVectorAggregatorTest.java

```diff
@@ -418,12 +419,12 @@ public void aggregateBatchWithRows()
     for (int i = 0; i < positions.length; i++) {
       Pair<Long, String> result = (Pair<Long, String>) target.get(buf, positions[i] + positionOffset);
       Assert.assertEquals(times[rows[i]], result.lhs.longValue());
-      Assert.assertEquals(VALUES[rows[i]], result.rhs);
+      Assert.assertEquals(NullHandling.nullToEmptyIfNeeded(VALUES[rows[i]]), result.rhs);
```

Comment:
Kind of, but not exactly. In the default value mode, the aggregator will now return the default value instead of null. Technically, this is the correct behavior (as opposed to the original behavior), however, in practice, the upstream callers will coerce null to the default value anyway, therefore it won't matter if null is returned by the aggregator.